app/assets/stylesheets/application.scss

```diff
@@ -22,6 +22,7 @@
 @import "users";
 @import "alliances";
 @import "mobile";
+@import "colors";
```

Comment:
Looks like you added $red to variables.scss. Could you just delete this file, and add color: $red; to whatever uses this class?